Ginger/GingerCoreCommonTest/VariableTests/ListVariableTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿#region License
+/*
+Copyright © 2014-2018 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Amdocs.Ginger.Common.Enums;
+using GingerCore.Variables;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+
+namespace GingerCoreCommonTest.VariableTests
+{
+    [TestClass]
+    [Level1]
+    public class ListVariableTests
+    {
+        List<string> lstTemp = new List<string>();
+
+        #region Default Class/Test Initialize Methods
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext TestContext)
+        {
+            //
+        }
+
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            //
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            // before every test
+        }
+
+        [TestCleanup]
+        public void TestCleanUp()
+        {
+            //after every test
+        }
+        #endregion
+
+        public ListVariableTests()
+        {
+            lstTemp.Add("Apple");
```

Comment:
Will be nice to have the list in each TC + use different values, and also check for empty list behavior, list with one item and list with special chars

Reply:
Added few more TC's